src/tcpconnect/tcpconnect.c

```diff
@@ -0,0 +1,108 @@
+#include <uapi/linux/ptrace.h>
```

Comment:
I would rename this file to `bpf.c`, naming it after the tool is redundant considering it's in its own folder in this project.